lib/board.rb

```diff
@@ -0,0 +1,49 @@
+class Board
+
+	attr_accessor :cells, :win_combinations
+
+	def initialize
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+	end
+
+
+	def reset!
+		@cells = [" ", " ", " ", " ", " ", " ", " ", " ", " "]
+		#The only way to get this test to pass was to put @cells.  Just "cells = ..." wouldn't work.
+		#cTried many different things here.
+
+	end
+
+	def display
+    	puts " #{cells[0]} | #{cells[1]} | #{cells[2]} "
+    	puts "-----------"
+    	puts " #{cells[3]} | #{cells[4]} | #{cells[5]} "
+    	puts "-----------"
+    	puts " #{cells[6]} | #{cells[7]} | #{cells[8]} "
+	end
+
+	def position(num)
+		cells[num.to_i-1]
+	end
+
+	def full?
+		!cells.include?(" ")
+	end
+
+	def turn_count
+		cells.select { |pos| pos == "X" || pos == "O" }.count
```

Comment:
Use the highest level and most expressive iterator possible to solve a problem. [`count`](http://ruby-doc.org/core-2.2.0/Array.html#method-i-count) on the cells is perfect and the rest of this method is not needed. Entire body could read:

```ruby
def turn_count
  cells.count { |cell| cell == "X" || cell == "O" }
end
```